tests/test_schema.yaml

```diff
@@ -800,6 +800,9 @@ schemas:
     - id
     - tenant_id
     - domain_id
+    required:
+    - tenant_id
+    - domain_id
```

Comment:
`required` will append `not null`. From gohan perspective, this is more proper way to define the properties.